app.py

```diff
@@ -0,0 +1,52 @@
+from flask import Flask, redirect, render_template, request
+from flask_cors import CORS 
+from flask_sqlalchemy import SQLAlchemy
+from controllers import urls as Urls
+
+
+#from werkzeug import exceptions
+
+app = Flask(__name__)
+CORS(app)
+
+POSTGRES = {
+    'user': 'debmon',
+    'pw': 'secretpassword',
+    'db': 'shortererl',
+    'host': 'localhost',
+    'port': '5430',
+}
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://%(user)s:%(pw)s@%(host)s:%(port)s/%(db)s' % POSTGRES
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+db = SQLAlchemy(app)
+
+@app.route('/', methods=['GET', 'POST'])
+def home():
+    if request.method == 'POST':
+        resp, code = Urls.create(request.form['original_url'])
+        return render_template('index.html', url=resp), code
+    else: 
+        return render_template('index.html')
+
+@app.route('/<short_url>')
+def new_link(short_url):
+        resp, code = Urls.redir(short_url)
+        return redirect(resp), code
+        
+
+
+
+
+'''
+@app.errorhandler(exceptions.NotFound)
```

Comment:
Was there a reason for commenting out the error handling? Would be good to add this back in, could import exceptions from werkzeug